### ISSUE TITLE
App crashes if you cover the camera.

### ISSUE THREAD

**Issue by MarkDaoust** [googlecodelabs/tensorflow-style-transfer-android#11]
If you cover the camera, the app crashes. 

It looks like this has something to do with the batch normalization.

Here's the output from `logcat`:

```
05-10 09:38:30.562 7402-7421/org.tensorflow.demo E/TensorFlowInferenceInterface: Failed to run TensorFlow inference with inputs:[input, style_num], outputs:[transformer/expand/conv3/conv/Sigmoid]
                                                                                 
                                                                                 --------- beginning of crash
05-10 09:38:30.565 7402-7421/org.tensorflow.demo E/AndroidRuntime: FATAL EXCEPTION: inference
                                                                   Process: org.tensorflow.demo, PID: 7402
                                                                   java.lang.IllegalArgumentException: input_max_range must be larger than input_min_range.
                                                                   	 [[Node: transformer/contract/conv1/Relu_eightbit_quantize_transformer/contract/conv1/InstanceNorm/batchnorm/add_1 = QuantizeV2[T=DT_QUINT8, mode="MIN_FIRST", round_mode="HALF_AWAY_FROM_ZERO", _device="/job:localhost/replica:0/task:0/device:CPU:0"](transformer/contract/conv1/InstanceNorm/batchnorm/add_1, transformer/contract/conv1/Relu_eightbit_min_transformer/contract/conv1/InstanceNorm/batchnorm/add_1, transformer/contract/conv1/Relu_eightbit_max_transformer/contract/conv1/InstanceNorm/batchnorm/add_1)]]
                                                                       at org.tensorflow.Session.run(Native Method)
                                                                       at org.tensorflow.Session.access$100(Session.java:48)
                                                                       at org.tensorflow.Session$Runner.runHelper(Session.java:298)
                                                                       at org.tensorflow.Session$Runner.run(Session.java:248)
                                                                       at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:228)
                                                                       at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:197)
                                                                       at org.tensorflow.demo.StylizeActivity.stylizeImage(StylizeActivity.java:590)
                                                                       at org.tensorflow.demo.StylizeActivity.access$1400(StylizeActivity.java:67)
                                                                       at org.tensorflow.demo.StylizeActivity$3.run(StylizeActivity.java:545)
                                                                       at android.os.Handler.handleCallback(Handler.java:751)
                                                                       at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                       at android.os.Looper.loop(Looper.java:154)
                                                                       at android.os.HandlerThread.run(HandlerThread.java:61)

```
